SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day2/exercise3/Exercise3.java

```diff
@@ -0,0 +1,34 @@
+package hackerrank.day2.exercise3;
+/**
+ * @author Sebastian Herrera (bermeosebas@gmail.com)
+ * @created 26/08/21
+ */
+
+import java.util.Scanner;
+
+public class Exercise3 {
+
+    public static final String PRINT_FORMAT = "================================";
+    public static final String FORMAT_ODER = "%-15s%03d\n";
+
+    /**
+     * Get information from user and formatting
+     */
+    public static void getScanInfo() {
+        Scanner sc = new Scanner(System.in);
+        int[] arrayInteger = new int[3];
+        String[] arrayString = new String[3];
+        for (int i = 0; i < 3; i++) {
```

Comment:
Even when all programing books include examples of for loops with var names like i, x, z, and so on, please avoid these.

HINT: (1 y 2) https://javarevisited.blogspot.com/2014/10/10-java-best-practices-to-name-variables-methods-classes-packages.html